sdk/python/tests/integration/registration/test_cli.py

```diff
@@ -128,6 +141,58 @@ def make_feature_store_yaml(project, test_repo_config, repo_dir_name: PosixPath)
     return yaml.safe_dump(config_dict)
 
 
+NULLABLE_ONLINE_STORE_CONFIGS: List[IntegrationTestRepoConfig] = [
+    IntegrationTestRepoConfig(),
+]
+if os.getenv("FEAST_IS_LOCAL_TEST", "False") != "True":
+    NULLABLE_ONLINE_STORE_CONFIGS.extend(
+        [
+            IntegrationTestRepoConfig(
+                provider="gcp",
+                offline_store_creator=BigQueryDataSourceCreator,
+                online_store="null",
+            ),
+            IntegrationTestRepoConfig(
+                provider="aws",
+                offline_store_creator=RedshiftDataSourceCreator,
+                online_store="null",
+            ),
+            IntegrationTestRepoConfig(
```

Comment:
I think you can just add this `IntegrationTestRepoConfig` object into `NULLABLE_ONLINE_STORE_CONFIGS` to begin with, and remove the empty `IntegrationTestRepoConfig`, and then only extend with the `gcp` and `aws` configs